CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-## v3.4.3 (unreleased)
```

Comment:
We also need an `(unreleased)` line with the next version number (v3.4.6).

Reply:
sounds good!